frameworks/helloworld/tests/test_overlay.py

```diff
@@ -73,16 +73,14 @@ def test_overlay_network():
     try:
         sdk_plan.wait_for_in_progress_recovery(PACKAGE_NAME, timeout_seconds=60)
         sdk_plan.wait_for_completed_recovery(PACKAGE_NAME, timeout_seconds=60)
+        sdk_tasks.check_running(PACKAGE_NAME, len(EXPECTED_TASKS))
     except TimeoutExpired:
         pass
 
     # test that the tasks are all up, which tests the overlay DNS
     framework_tasks = [task for task in shakedown.get_service_tasks(PACKAGE_NAME, completed=False)]
     framework_task_names = [t["name"] for t in framework_tasks]
 
-    for expected_task in EXPECTED_TASKS:
```

Comment:
This changes what is being tested quite significantly. The check that has been moved into the `try`-`except` block now only checks for the number of tasks and not specific tasks. The question is what the expected behaviour is.